chain/chain/src/validate.rs

```diff
@@ -78,39 +60,6 @@ pub fn validate_chunk_proofs(
     Ok(true)
 }
 
-/// Validates that the given transactions are in proper valid order.
-/// See <https://nomicon.io/ChainSpec/Transactions.html#transaction-ordering>
-pub fn validate_transactions_order(transactions: &[SignedTransaction]) -> bool {
```

Comment:
Could you remind me why doesn't transaction ordering matter any more and where was this originally used? I remember it was rendered vestigial after relaxed chunk validation?

Reply:
We stopped using it when we introduced relaxed chunk validation.  And I confirmed with Bowen that this code is no longer needed.